.github/actions/constellation_create/action.yml

```diff
@@ -124,17 +124,28 @@ runs:
       run: |
         yq eval -i '(.internalLoadBalancer) = true' constellation-conf.yaml
 
-    # Uses --force flag since the CLI currently does not have a pre-release version and is always on the latest released version.
-    # However, many of our pipelines work on prerelease images. Thus the used images are newer than the CLI's version.
-    # This makes the version validation in the CLI fail.
-    - name: Constellation create
+    - name: Show Cluster Configuration
       shell: bash
       run: |
         echo "Creating cluster using config:"
         cat constellation-conf.yaml
         sudo sh -c 'echo "127.0.0.1 license.confidential.cloud" >> /etc/hosts' || true
+
+    # Uses --force flag since the CLI currently does not have a pre-release version and is always on the latest released version.
+    # However, many of our pipelines work on prerelease images. Thus the used images are newer than the CLI's version.
+    # This makes the version validation in the CLI fail.
```

Comment:
Does this still hold true?
We don't need to test in this PR, but I'm wondering if we can drop the force flag for the CI by now